addons/core/translations/en-us.yaml

```diff
@@ -1,10 +1,14 @@
 auth:
   titles:
     no-scopes: No Scopes Available
+    flow-success: Successful authentication
+    flow-error: Could not authenticate to provider
```

Comment:
Could we eliminate `provider`? As we are using `authentication` instead of identifying oidc or provider in this route, we could simplify this:
> Could not authenticate ~~to provider~~
> Something went wrong while authenticating ~~with the provider~~.  Please close this window and try again.  If the problem persists, notify your administrator.

Reply:
+1 vote on @gsusmi suggestions.  Most users will have no context to understand what a provider is.

Reply:
Good point!  I even intentionally made the translation keys generic in anticipation of (get excited) SAML.  And yet the content is not quite generic.  Will update.